doc/release-notes/5.3-release-notes.md

```diff
@@ -0,0 +1,96 @@
+
+### Payara 5.2020.6 Required
+
+Some changes in this release require an upgrade to Payara 5.2020.6.
+
+Instructions how to update can be found in the
+[Payara documentation](https://docs.payara.fish/community/docs/5.2020.6/documentation/user-guides/upgrade-payara.html)
+
+### New Banner API, Obsolete DB Settings
+
+The functionality previously provided by the DB settings :StatusMessageHeader and ::StatusMessageText is no longer supported and is now provided through the Manage Banner Messages API. Learn more in the [API Guide](https://guides.dataverse.org/en/5.3/api/).
+
+### New Database Settings and JVM Options
+
+No new DB Settings or JVM Options have been added in this release. See above note about obsolete options.
+
+## Notes for Tool Developers and Integrators
+
+### Experimental Auxiliary File Support
+
+Experimental endpoints have been added to allow auxiliary files to be added to datafiles. These auxiliary files can be deposited and accessed via API. Later releases will include options for accessing these files through the UI. For more information, see the Auxiliary File Support section of the [Developer Guide](https://guides.dataverse.org/en/5.3/developers/).
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.3 Milestone](https://github.com/IQSS/dataverse/milestone/93?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.3/installation/).
+
+## Upgrade Instructions
+
+0\. These instructions assume that you've already successfully upgraded from Dataverse 4.x to  Dataverse 5 following the instructions in the [Dataverse 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0).
+
+1\. Undeploy the previous version.
+
+- `<payara install path>/bin/asadmin list-applications`
+- `<payara install path>/bin/asadmin undeploy dataverse<-version>`
+
+(where `<payara install path>` is where Payara 5 is installed, for example: `/usr/local/payara5`)
+
+2\. Upgrade to Payara 5.2020.6 following the steps in the [Payara documentation](https://docs.payara.fish/community/docs/5.2020.6/documentation/user-guides/upgrade-payara.html)
+
+3\. Stop payara and remove the generated directory, start.
+
+- `service payara stop`
+- remove the generated directory:
+`rm -rf <payara install path>/payara/domains/domain1/generated`
+- `service payara start`
+
+4\. Deploy this version.
+
+- `<payara install path>/bin/asadmin deploy dataverse-5.3.war`
+
+5\. Restart payara
+
+- `service payara stop`
+- `service payara start`
+
+6\. Reset the EJB timer database back to default:
+
+- `<payara install path>/asadmin set configs.config.server-config.ejb-container.ejb-timer-service.timer-datasource=jdbc/__TimerPool`
```

Comment:
I think this needs a /bin/

Reply:
added in https://github.com/IQSS/dataverse/commit/232751af9f6be3c2ffcaee01c655aefe53a0d658